grafana-plugin/src/containers/RotationForm/RotationForm.tsx

```diff
@@ -699,18 +707,38 @@ export const RotationForm = observer((props: RotationFormProps) => {
     </>
   );
 
+  function onResize() {
+    onHide();
```

Comment:
why not call `onHide` directly without creating proxy fn as it does only one thing?

Reply:
Hmm, the passed prop could change reference and you no longer remove the correct associated handler, that was my thinking back then

Reply:
Ok, we can leave like that, I missed it's passed as a prop 👍